app/src/main/java/com/example/parkingsystem_kotlin/ParkingReservationActivity.kt

```diff
@@ -30,7 +35,8 @@ class ParkingReservationActivity : AppCompatActivity() {
         setContentView(binding.root)
         presenter = ParkingReservationPresenter(
             ParkingReservationView(binding, this),
-            ParkingReservationModel()
+            ParkingReservationModel(),
+            parkingSize
```

Comment:
please pass this parameter in a presenter.init(parkingSize) method, not in the constructor